From 053c07b5f2e3cb0981c857973d962f0fc023b3c2 Mon Sep 17 00:00:00 2001
From: Arne Coucheron <arco68@gmail.com>
Date: Sat, 1 Jul 2017 09:58:41 +0200
Subject: [PATCH 30/31] Increase to 5 rows on some device profiles

These now have lower icon sizes than originally,
so 5 rows should fit perfectly.

Change-Id: Ie39b1ce658882ce9fae95dd4e10021700c07856c
---
 res/xml/device_profiles.xml | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/res/xml/device_profiles.xml b/res/xml/device_profiles.xml
index 83acbdfac..721a13246 100644
--- a/res/xml/device_profiles.xml
+++ b/res/xml/device_profiles.xml
@@ -81,7 +81,7 @@
         launcher:name="Nexus S"
         launcher:minWidthDps="296"
         launcher:minHeightDps="491.33"
-        launcher:numRows="4"
+        launcher:numRows="5"
         launcher:numColumns="4"
         launcher:numFolderRows="4"
         launcher:numFolderColumns="4"
@@ -96,7 +96,7 @@
         launcher:name="Nexus 4"
         launcher:minWidthDps="335"
         launcher:minHeightDps="567"
-        launcher:numRows="4"
+        launcher:numRows="5"
         launcher:numColumns="4"
         launcher:numFolderRows="4"
         launcher:numFolderColumns="4"
@@ -111,7 +111,7 @@
         launcher:name="Nexus 5"
         launcher:minWidthDps="359"
         launcher:minHeightDps="567"
-        launcher:numRows="4"
+        launcher:numRows="5"
         launcher:numColumns="4"
         launcher:numFolderRows="4"
         launcher:numFolderColumns="4"
-- 
2.17.1

